services/movie.py

```diff
@@ -0,0 +1,45 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+    if not (genres_ids or actors_ids):
+        return Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        return Movie.objects.filter(
+            actors__in=actors_ids,
+            genres__in=genres_ids
+        )
+
+    if not actors_ids:
+        return Movie.objects.filter(genres__in=genres_ids)
+
+    return Movie.objects.filter(actors__in=actors_ids)
```

Comment:
You can apply a few filters to one queryset. So you can simplify your code
```suggestion
    queryset = Movie.objects.all()

    if actors_ids:
        queryset = queryset.objects.filter(actors__in=actors_ids)
        
    if genres_ids:
        queryset = queryset.objects.filter(genres__in=genres_ids)

    return queryset
```

Reply:
Thanks for the advice, I improved this assignment a little :)